static/src/js/actions/subscriptions.js

```diff
@@ -116,6 +117,13 @@ export const createSubscription = (name, subscriptionType) => async (dispatch, g
   }
   params.query = subscriptionQuery
 
+  if (!name) {
+    params.name = formatDefaultSubscriptionName(
```

Comment:
We could potentially use [nullish logical assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_nullish_assignment) here (just have to make sure we are using the right babel plugin)

```js
params.name ??= formatDefaultSubscriptionName(
    getCollectionSubscriptionQueryObj(state),
    subscriptionType
)
```

Reply:
Nice idea. I'll give it a shot.